lstchain/tests/test_lstchain.py

```diff
@@ -44,9 +50,28 @@ def test_r0_to_dl1():
     r0_to_dl1(infile, custom_config=standard_config, output_filename=dl1_file)
 
 
+@pytest.mark.private_data
+def test_r0_to_dl1(tmp_path):
+    from lstchain.reco.r0_to_dl1 import r0_to_dl1
+
+    r0_to_dl1(
+        test_r0_path,
+        output_filename=tmp_path / ('dl1_' + test_r0_path.stem + '.h5'),
+        custom_config=standard_config,
+        pedestal_path=test_drs4_pedestal_path,
+        calibration_path=test_calib_path,
+        time_calibration_path=test_time_calib_path,
+        pointing_file_path=test_drive_report,
+        ucts_t0_dragon=None,
+        dragon_counter0=None,
+        ucts_t0_tib=None,
+        tib_counter0=None,
+    )
```

Comment:
Don't you need any assertion statement here?

Reply:
For now I just wanted to see that it runs without error.